src/basic_potentials.jl

```diff
@@ -119,29 +119,28 @@ function pairwise_electrostatic_acceleration!(dv,
     p::ElectrostaticParameters,
     pbc::BoundaryConditions)
 
-    force = @SVector [0.0, 0.0, 0.0];
+    force = @SVector [0.0, 0.0, 0.0]
     ri = @SVector [rs[1, i], rs[2, i], rs[3, i]]
     @inbounds for j = 1:n
         if !in(j, exclude[i])
             rj = @SVector [rs[1, j], rs[2, j], rs[3, j]]
-
-            (rij, rij_2, success) = apply_boundary_conditions!(ri, rj, pbc, p.R2)
-            if success
-                force += qs[j] * rij / norm(rij)^3
+            rij, r, r2 = get_interparticle_distance(ri, rj, pbc)
+            if r2 < p.R2
+                force += qs[j] * rij / (r*r2)
```

Comment:
is this all of the time or 128? May want to `ifelse`?

Reply:
`ifelse` didn't help here in my testing, which isn't surprising - that branch will be false more often than not in most simulations. The standard way to improve performance here is to replace the whole approach with cached neighbor lists or PME type approaches, but those are way beyond the scope of this PR.